### PR TITLE
refactor(client/dead): general cleanup

### PR DESCRIPTION
**Pull request by Manason**
**Describe Pull request**
- simplifying code, re-using existing functions from other files, extracting functions, getting rid of nesting.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [no] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

File: client/dead.lua [hunk 1]
```diff
@@ -1,68 +1,44 @@
...
 
--- Functions
-function OnDeath()
-    if not isDead then
-        isDead = true
-        TriggerServerEvent("hospital:server:SetDeathStatus", true)
-        TriggerServerEvent("InteractSound_SV:PlayOnSource", "demo", 0.1)
-        local player = PlayerPedId()
+local function playDeadAnimation(player)
+    if cache.vehicle then
```
Comment (BerkieBb) [inline client/dead.lua:5]: Reverse this if statement to reduce nesting

File: client/laststand.lua [hunk 2]
```diff
@@ -30,11 +30,11 @@ local function GetClosestPlayer()
     return closestPlayer, closestDistance
 end
 
-local function waitForPlayerToStopMoving(ped)
+function WaitForPlayerToStopMoving(ped)
```
Comment (BerkieBb) [inline client/laststand.lua]: Maybe more logical to call it `WaitForPedToStopMoving`

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: client/dead.lua (hunk 1 above)
Comment (Manason) [reply client/dead.lua:5]: Bit confused by what you mean. We could do a return in the first if and get rid of the else, but I don't that would improve the readability here.

Comment (BerkieBb) [reply client/dead.lua:5]: I didn't see the else, it's fine then

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**